habs_portal/static/css/stations.css

```diff
@@ -10,6 +10,11 @@
   height: 100%;
 }
 
+.plotting-row {
+  height: 300px;
```

Comment:
Ok. Related: I was thinking about splitting the plotting panel down the middle. Plot on top, table on bottom. This, http://jsfiddle.net/gaby/Bek9L/1779/ but horizontally. If we won't support dynamic graph resize I won't go this route (I'll make the table responsive). But the first thing boss man will want to do is resize the graph.